src/components/Bar.svelte

```diff
@@ -138,18 +134,61 @@
     return clone
   }
 
+  /**
+   * Helper function to update apps in CONFIG tree
+   * @param  {Object}           config the JSON CONFIG tree source
+   * @return {Promise(boolean)} a valve that allow to trigger update or not
```

Comment:
Nope, it returns a `valve` in functionnal term: to boolean that allow to block or pass a process later :smiley: (see https://github.com/cozy/cozy-bar/pull/28/files/6ff3b3fa99fe41f76933bf0f945073caac793954#diff-c8a245b0e389919c587f61c8d66df421R230 for some kind of use)